src/config/chainEndpoints.ts

```diff
@@ -136,9 +136,8 @@ export const getProviderIndex = (chain: ASTAR_CHAIN) => {
       return endpointKey.ASTAR;
     case 'Shiden':
       return endpointKey.SHIDEN;
-    case 'Shibuya Testnet':
-      return endpointKey.SHIBUYA;
+
     default:
-      return endpointKey.ASTAR;
+      return endpointKey.SHIBUYA;
```

Comment:
Is there a reason why we changed the default from Astar to Shibuya?

Reply:
I'd like to return `Shibuya` for the local node network. Otherwise, it conflicts with `xcm hooks` and `local node network` whenever users connect to the local node. Also, I want to avoid invoking mainnet logic such as fetching price (USD) data whenever users connect to the local node.

![image](https://user-images.githubusercontent.com/92044428/171587033-d0e902d8-9ac5-4fe2-ac4f-e1929bed79f0.png)

related: https://github.com/AstarNetwork/astar-apps/issues/423